components/Article.jsx

```diff
@@ -0,0 +1,46 @@
+
+import { useParams } from 'react-router-dom';
+import { useState, useEffect } from 'react'
+import axios from "axios";
+
+
+const Article = ()=>{
+    const [article, setArticle] = useState({});
+    const {article_id } = useParams()
+
+
+    console.log("topman");
+
+
+    useEffect(() => {
+        axios
+          .get(`https://nc-news-api-4.onrender.com/api/articles/${article_id}`)
```

Comment:
Our axios request should really be happening in a separate utils file. You can call this api.js and put all your api calls in there 👍